### PR TITLE
When a job fails or is canceled, update the large image item to reflect that.

### PR DESCRIPTION
**Pull request by manthey**
Send a file to the processing job rather than an item.  This allows processing an image from an item that has more than one file.

This change was prompted by trying to convert a file with a name that differed from its containing item in a non-filesystem assetstore.

### CODE REVIEW

**Comment by manthey** [comment]
This addresses issue #49.

**Comment by manthey** [comment]
@zachmullen (or anyone else), I'd love to see this reviewed and merged.

The switch in the tests to use rabbitmq as the broker should allow all tests to pass in the vagrant development environment.

File: server/base.py [hunk 1]
```diff
@@ -50,6 +51,44 @@ def _postUpload(event):
...
+    job = event.info['job']
+    meta = job.get('meta', {})
+    if (meta.get('creator') != 'large_image' or not meta.get('itemId') or
+            meta.get('task') != 'createImageItem'):
+        return
+    status = job['status']
+    if status not in (JobStatus.ERROR, JobStatus.CANCELED, JobStatus.SUCCESS):
+        return
+    item = ModelImporter.model('item').load(meta['itemId'], force=True,
+                                            exc=False)
```
Comment (zachmullen) [inline server/base.py]: `exc=False` is the default

**Comment by zachmullen** [comment]
LGTM other than the one nit

**Pull request merged**